import_export/admin.py

```diff
@@ -116,6 +116,21 @@ def process_import(self, request, *args, **kwargs):
 
             return self.process_result(result, request)
 
+        res_kwargs = self.get_import_resource_kwargs(request, form=confirm_form, *args, **kwargs)
+        resource = self.get_import_resource_class()(**res_kwargs)
+
+        context = self.get_import_context_data()
+        context.update(self.admin_site.each_context(request))
+
+        context['title'] = _("Import confirmation")
```

Comment:
Typo: (...) = _("Import confirmation")  -> (...) = ("Import confirmation")

Reply:
Thanks for testing.  I'm not seeing the typo - please can you clarify?  The leading `_` char is for [translations](https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#standard-translation).

Reply:
I get a "_" is not defined error

Reply:
This PR / branch is out-of-date, please use the [release-3-x](https://github.com/django-import-export/django-import-export/tree/release-3-x) branch.